modules/dev2plus/hw_01/src/Node/Node.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Node of the tree.
+ *
+ * @public
+ * @constructor
+ * @param {Number|String} data Value of the node.
+ * @param {Node} left Left sibling.
+ * @param {Node} right Right sibling.
+ * @param {Node} parent Parent of the node.
+ */
+class Node {
+  constructor(value, left = null, right = null, parent = null) {
+    this.data = value;
+    this.left = left;
+    this.right = right;
+    this.parent = parent;
+  }
+
+  getValue() {
+    return {
+      value: this._value,
```

Comment:
what is `this._value` ? 
i think returning the value is enough here, you don't need to wrap it

Reply:
You are right i read here first time: https://mgechev.github.io/javascript-algorithms and I reused it. I have a couple of mistakes, thx :)